action.yml

```diff
@@ -159,10 +165,24 @@ runs:
       run: |
         set -e
         echo -e "\e[32mCleaning up...\e[0m"
-        KNOWN_HOSTS_FILE=~/.ssh/known_hosts.sshupload
```

Comment:
Why it wasn't kept as variable if it's executed twice?

Reply:
which variable, in env: ? it's a different one. This is a file that's created in previous step, and it's used in each step exactly once. having a same-named variable inside the step that has a different value than above just obfuscates the logic even more

Reply:
This whole action steps are kinda cryptic, but I don't have a better idea to do it.